docs/pages/documentation/3-querying-with-graql/graql-shell.md

```diff
@@ -24,12 +24,12 @@ You can optionally pass arguments when starting the Graql shell, as follows:
 
 | Long Option   | Option   | Description                                      | Note |
 | ------------- | -------- | ------------------------------------------------ | |
-| `--batch <arg>`     | `-b`     | A path to a file containg a query to batch load. | The REPL does not open. <br/> The resulting knowledge base is automatically committed. |
+| `--batch <arg>`     | `-b`     | A path to a file containg a query to batch load. | The REPL does not open. <br/> The resulting knowledge base is automatically committed. <br/> **Note:** In this mode inference is disabled to ensure data load is consistent regardless of load order  |
 | `--execute <arg>`     | `-e`     | A query to execute.                              | The REPL does not open. <br/> The resulting knowledge base is automatically committed. |
 | `--file <arg>`      | `-f`     | A path to a file containg a query to execute.    | The REPL does not open. <br/> The resulting knowledge base is automatically committed. |
 | `--help`      | `-h`     | Print usage message.                             | |
 | `--keyspace <arg>`  | `-k`     | The keyspace of the knowledge base.                 | |
-| `--infer`     | `-n`     | Perform inference on results.                    | Reasoning is not enabled by default at present, although as Grakn develops, we expect that to change.|
+| `--infer`     | `-n`     | Perform inference on results.                    | |
```

Comment:
I would actually skip this change, because we're making a similar (but not identical one) to `stable`. One less merge conflict for later 👍